recipes/ucrt/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "ucrt" %}
+{% set version = "10.0.20348.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://software-download.microsoft.com/download/sg/20348.1.210507-1500.fe_release_WindowsSDK.iso
+  sha256: a0cdc3fc9db69084c8dd8c09ee153c1a168366ac4dfc3d890abbf0bf17e635dc
+
+build:
+  number: 0
+  script:
+    - mkdir -p $PREFIX/Library/bin
+    - 7z x 20348.1.210507-1500.fe_release_WindowsSDK.iso -aoa
+    - msiextract -C tmp "Installers/Universal CRT Redistributable-x86_en-us.msi"
+    - cp "tmp/Program Files/Windows Kits/10/Redist/{{ version }}/ucrt/DLLs/x64/"* $PREFIX
+    - cp "tmp/Program Files/Windows Kits/10/Redist/{{ version }}/ucrt/DLLs/x64/"* $PREFIX/Library/bin
+
+  # This is supposed to be a win-64 package only, but it is noarch: generic because
+  # I don't want to write this in cmd.
+  noarch: generic
```

Comment:
Happy to do this for you. With the above commands, I should be able to write this with CMD and Windows compatible dependencies.

Also happy to be added as a maintainer.

Reply:
Thanks @xhochy

Reply:
Feel free to push to this branch.